examples/mockups/company.html

```diff
@@ -0,0 +1,481 @@
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                  <a class="ui avatar" href=""><img class="ui avatar image" src="./images/team/default-avatar.svg"><span>Username</span></a>
+                </div>
+              </div>
+            </div>
+
+          </div>
+        </div>
+
+      </div>
+    </section>
+
+
+    <!-- Resources -->
+    <section class="ui vertical light segment padded" id="resources">
+
+      <div class="ui container">
+        <div class="ui grid stackable three column">
+
+          <div class="column">
+            <div class="ui basic segment">
+              <h3 class="ui header huge">Our Blog</h3>
+              <p>
+                This is where we post all the week to week progress updates, announcements and news. If it's something meriting extra visibility we might also display in the website or inside the app, depending of the specific scope but, if it's written somewhere, you can always be sure to also find it in <a href="https://blog.readthedocs.com/">our blog</a>.
```

Comment:
I'm not sure 🤷 a card view blog post could work too.

On the other hand, keeping in mind the objective of explaining the product clearly on  this frontpage, to new and potential users, maybe a blog post here would be repeating the announcement sections at the top (if the post merits enough highlight, it will be there) and would be creating more cluster without adding anything new.